AUTHORS.rst

```diff
@@ -10,4 +10,6 @@ Development Lead
 Contributors
 ------------
 
-None yet. Why not be the first?
+* Steve Jalim <stevejalim@mozilla.com>
```

Comment:
You sure you want your email this public?

Reply:
Not too fussed, but swapped for a Firefox Relay address - will be interesting to see what comes in